app/models/email_subscription.rb

```diff
@@ -56,4 +58,18 @@ def deactivate!
   ensure
     update!(email_alert_api_subscription_id: nil)
   end
+
+  def send_transition_checker_onboarding_email!(email)
+    return unless email_alert_api_subscription_id
+    return unless name == "transition-checker-results"
```

Comment:
Do we think we'll have others of these?
Wondering if we should put this in a const at the top or something and then do:
`return unless ALLOWED_NAMES.include?(name)`

Could always add that when we've got the second service though?

Reply:
Yeah, I think we should leave any attempt at generalisation until we have another thing which has an onboarding email.  We can't predict what we'll need.